src/reps/rep-utils.js

```diff
@@ -167,5 +230,8 @@ module.exports = {
   parseURLParams,
   parseURLEncodedText,
   getFileName,
-  getURLDisplayString
+  getURLDisplayString,
+  isGripSelectableInInspector,
+  getFlattenedGrips,
```

Comment:
Is getFlattenedGrips used outside of this file? If not I wouldn't export it as it's pretty specific.